### PR TITLE
 vm::var improved, cleanup

### PR DESCRIPTION
**Pull request by Nekotekina**

### CODE REVIEW

**Comment by vlj** [comment]
Why did you remove tambry's coverity fixes?

File: rpcs3/stdafx.h [hunk 1]
```diff
@@ -165,12 +167,15 @@ template<typename T1, typename T2, typename T3 = const char*> struct triplet_t
...
 #define IS_INTEGER(t) (std::is_integral<t>::value || std::is_enum<t>::value || std::is_same<std::decay_t<t>, u128>::value)
 #define IS_BINARY_COMPARABLE(t1, t2) (IS_INTEGER(t1) && IS_INTEGER(t2) && sizeof(t1) == sizeof(t2))
+#define CHECK_ASSERTION(expr) if (expr) {} else throw EXCEPTION("Assertion failed: " #expr)
+#define CHECK_SUCCESS(expr) if (s32 _r = (expr)) throw EXCEPTION(#expr " failed (0x%x)", _r)
 
 template<typename T> struct id_traits;
 
 #define _PRGNAME_ "RPCS3"
-#define _PRGVER_ "0.0.0.6"
+#define _PRGVER_ "0.0.0.5"
```
Comment (Zangetsu38) [inline rpcs3/stdafx.h]: What revert this ? Version "0.0.0.6" no 0.0.0.5

**Comment by Syphurith** [comment]
@vlj @Zangetsu38 Since this seems to affect many files, he might just clicked use my files to solve all those conflicts. If those meaningful edits are not so much, you may adjust all those again to approve your rank in the contributors list of the repo itself (orz). Well.

File: rpcs3/stdafx.h (hunk 1 above)
Comment (Nekotekina) [inline rpcs3/stdafx.h]: Yeah, forgot this.

**Comment by Nekotekina** [comment]
@vlj I will remove revert commit once I understand the situation better

File: rpcs3/stdafx.h (hunk 1 above)
Comment (Zangetsu38) [inline rpcs3/stdafx.h]: ok

**Pull request merged**